src/interpreter/ByteCode.h

```diff
@@ -1472,6 +1475,68 @@ class CallFunctionInWithScope : public ByteCode {
 #endif
 };
 
+class GeneratorComplete : public ByteCode {
+public:
+    GeneratorComplete(const ByteCodeLOC& loc, bool isThrow = false)
+        : ByteCode(Opcode::GeneratorCompleteOpcode, loc)
+        , m_isThrow(isThrow)
+    {
+    }
+
+    bool m_isThrow;
+
+#ifndef NDEBUG
+    void dump(const char* byteCodeStart)
```

Comment:
Argument is not used, please remove it.

Reply:
It cannot be removed, due to the [branch instructions](https://github.com/pando-project/escargot/blob/master/src/interpreter/ByteCode.cpp#L54).